sos/cleaner/mappings/hostname_map.py

```diff
@@ -88,6 +119,7 @@ def sanitize_item(self, item):
             # obfuscate the short name
             ob_hostname = self.sanitize_short_name(hostname)
             ob_domain = self.sanitize_domain(domain)
+            self.dataset[item] = ob_domain
```

Comment:
`ob_domain` only, or also `ob_hostname` (like the return value)? I need to review it first, I might be wrong..

Reply:
This specifically was for getting subdomains added to the dataset in addition to the original domain sourced during initialization. Short names are/should be handled elsewhere.

In my tests now, I'm not seeing the hostname persist in the archives anywhere, save for local rpm builds since we're currently skipping all rpm files for all parsers (which is a separate issue to this one).